### ISSUE TITLE
STFT and exploding training

### ISSUE THREAD

**Issue by eagomez2** [felixkreuk/HideAndSpeak#3]
Hi Felix,

First of all thanks for sharing this repo, it's an amazingly interesting work. Since the datasets you used to my knowledge are paid and only at 16k and 8k, respectively, I wanted to train your network using the VCTK dataset. For training it, I only chose small files with high activity of the trimmed version of it to avoid long sections of silence. However, when I do so, the training goes well for a few epochs and the message becomes intelligible very fast, but then the loss (I am using L1) explodes to a value order of magnitudes higher as shown [here](https://drive.google.com/file/d/1Ob1ULj9zvC--4N2s5nMop9Ob06Z0kClE/view?usp=sharing). Have you observed this pattern during your trainings?

As a second question, I wanted to know what you used [your own STFT](https://github.com/felixkreuk/HideAndSpeak/blob/master/stft/stft.py) implementation as opposed to [the torchaudio ones](https://pytorch.org/audio/stable/_modules/torchaudio/transforms.html#Spectrogram) that allow backprop.

Thanks!
